.github/workflows/pr_build.yml

```diff
@@ -14,7 +14,9 @@ env:
   RELEASE_TYPE: PR
   RELEASE_NUMBER: ${{ github.event.number }}
   VERSION_CODE: ${{ github.event.number }}
-
+  # Sentry Crash Reporting
+  CMAKE_BACKTRACE_URL: https://sentry.vircadia.dev/api/2/minidump/?sentry_key=dbf6000201a442e0b5462b63e18ee6b1
```

Comment:
is this a security concern?  should this key be kept private?  also, should this be added to the master_builds.yaml too?

Reply:
Yes, it'll be helpful if they are on both. I thought about security and... Well, I mean it would only be security by obsfucation. So I suppose we can do it that way? If we make it a GHA env variable.